core/src/main/java/hudson/XmlFile.java

```diff
@@ -185,8 +187,19 @@ private Object unmarshal(Object o, boolean nullOut) throws IOException {
     }
 
     public void write( Object o ) throws IOException {
+        write(o, true);
+    }
+
+    /**
+     * @param force Whether or not to flush the page cache to stable storage with {@link
```

Comment:
I feel like rather than an arg to an overload of `write`, this would be more natural as a constructor parameter, or a fluent-style method

```java
public XmlFile withoutForce() {
    force = false;
    return this;
}
```